### PR TITLE
.mailmap: update dekarl's info.

### PR DESCRIPTION
**Pull request by mkrautz**

### CODE REVIEW

**Comment by mkrautz** [comment]
@hacst @Kissaki @Natenom @fwaggle PTAL

File: .mailmap [hunk 1]
```diff
@@ -105,3 +105,6 @@ Derrick Dymock <actown@gmail.com> Derrick Dymock <derrick@puppetlabs.com>
 
 # Alias of Joel Kees <joelkees@gmail.com>
 Joel Kees <joelkees@gmail.com> Joeliam <joelkees@gmail.com>
+
+# Alias of Karl Dietz <dekarl@users.sourceforge.net>
```
Comment (hacst) [inline .mailmap]: Shouldn't this be the "spaetfruehstuecken" one (assuming that's the new cannonical one)?

**Comment by hacst** [comment]
So the comment wasn't backwards?

**Comment by mkrautz** [comment]
Oops. Yes it was. I knew I shouldn't have landed the commits that late. :-)

**Comment by Kissaki** [comment]
Fixed in 72cd86c966f309b7ff29b504132829dbeaf30ef9

**Pull request merged**